printer_mods/Phalanx/Other-V2-Idlers/README.md

```diff
@@ -0,0 +1,55 @@
+# "Other" V2 Idlers
+ Alternative Idlers that allow the usage of "Full-Figured" toolheads without the 10mm loss of travel at the front corners of the bed.
+
+These idlers were developed with the goal to allow the usage of wider toolheads with the Voron V2, better align the tensioning screw to the centerline of the idler shaft, add guides to the parts to allow them to interlock and provide a robust tensioning system, all while streamlining the process of making fine adjustments to tensioning. 
+This design iteration has been running on my machine for 50+ hrs with no issues or print quality degradation.  
+
+The idler design is ambidextrous to allow simplified batch printing. All parts are converted to stl in optimal printing orientation with quantities specified per Voron standards: 
+
+![image](https://user-images.githubusercontent.com/65414930/123886695-a1619680-d915-11eb-8a33-212a3cb1b8ef.png)
+
+
+# Print Settings:
+All parts should be printed to standard Voron print settings:
+
+![image](https://user-images.githubusercontent.com/65414930/123886729-b63e2a00-d915-11eb-81b5-52c1082b56c9.png)
+
+Note: If your printer is not properly tuned i.e. printing dimensionally-accurate parts free from functional defects, the slide mechanism WILL NOT work proerly. 
+
+# BOM:
+2 ea. 5mm OD x 45mm L pins - may need to be polished down to fit the bearings depending on source. 
```

Comment:
Can you convert this a markdown list, so it displays one per line?  Right now it looks like this:

<img width="1038" alt="Screen Shot 2021-07-03 at 14 43 11" src="https://user-images.githubusercontent.com/69389593/124364048-03b7e100-dc0d-11eb-81dd-20b8f43dbc18.png">

You can make it a list by preceding each line with a `- `, e.g:

```
# BOM
- 2 ea. 5mm OD x 45mm L pins - may need to be polished down to fit the bearings depending on source.
- 4 ea. F695 bearings
```

which would look like:

# BOM
- 2 ea. 5mm OD x 45mm L pins - may need to be polished down to fit the bearings depending on source.
- 4 ea. F695 bearings